astroid/brain/brain_dataclasses.py

```diff
@@ -330,7 +342,11 @@ def _looks_like_dataclass_field_call(node: Call, check_scope: bool = True) -> bo
     return inferred.name == FIELD_NAME and inferred.root().name in DATACLASS_MODULES
 
 
-def _get_field_default(field_call: Call) -> Tuple[str, Optional[NodeNG]]:
+def _get_field_default(
+    field_call: Call,
+) -> Union[
+    None, Tuple[Literal["default"], NodeNG], Tuple[Literal["default_factory"], Call]
+]:
```

Comment:
I could be worth creating an Alias for it. Even if for nothing else than readability.